### PR TITLE
feat: support anchor commitments in tests

### PR DESCRIPTION
**Pull request by openoms**
fixes: #1516

### CODE REVIEW

File: src/services/lnd/utils.ts [hunk 1]
```diff
@@ -115,8 +115,19 @@ export const lndBalances = async (lnd: AuthenticatedLnd) => {
...
 
+  // adds 330 sats for every selfInitiatedChannel even if not all channels use anchor
+  const { channels } = await getChannels({ lnd })
+  const selfInitiatedChannels = channels.filter(
+    ({ is_partner_initiated }) => is_partner_initiated === false,
+  )
+  let anchorBalance = 0
+  for (let selfInitiatedChannel of selfInitiatedChannels) {
+    anchorBalance += 330
+  }
```
Comment (nicolasburtey) [inline src/services/lnd/utils.ts:126]: Const satsAnchorOutput = 330
Let anchorBalance = selfInitiatedChannel * satsAnchorOutput

**Review by nicolasburtey — CHANGES_REQUESTED** [review]
You don't need a loop here

**Comment by nicolasburtey** [comment]
Also check code is failing. Make sure to run 'make check-code' and debug any issue that would be shown

File: src/services/lnd/utils.ts (hunk 1 above)
Comment (openoms) [reply src/services/lnd/utils.ts:126]: I have used a loop because selfInitiatedChannels is an array.

Comment (nicolasburtey) [reply src/services/lnd/utils.ts:126]: Ok then you can just take the length of the array

Comment (openoms) [reply src/services/lnd/utils.ts]: thanks for the suggestion, made the change in [be1f13c](https://github.com/GaloyMoney/galoy/pull/1520/commits/be1f13c8748a814e458375c2bde6117898cf140b) and tests are passing locally

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**